pkg/github/github.go

```diff
@@ -156,6 +156,11 @@ func CollectComponents(repoConfig repositories.Config, httpClient *http.Client)
 				return nil, err
 			}
 
+			// No new version between previous and current suite - discard change notes
+			if repo.Version == repo.AfterVersion {
+				component.Changelogs = nil
+			}
```

Comment:
I think this change will erroneously affect `unreleased` output type and we may need to check that the `changelog` type is accurately generated since it uses different logic paths. The desired outcome would be that `changelog`, `release` and `web-release` have the omission but for `unreleased` output we still want to include that changelog. I think we my want to change our integration test data to check that this is exactly what happens.